src/js/main.js

```diff
@@ -67,12 +67,12 @@ var SupportKit = Marionette.Object.extend({
         }
 
 
-        if(/lebo|awle|pide|obo|rawli/i.test(navigator.userAgent)) {
+        if (/lebo|awle|pide|obo|rawli/i.test(navigator.userAgent)) {
```

Comment:
Unrelated to the PR, but still a good question. My guess is that it's probably as good as what Chris had. Instead of injecting the whole widget it only injects the link. The regex should cover any user agent that has those string (Googlebot, crawler, spider, robot, crawling) which covers mostly all civilized crawlers.